libsolidity/analysis/ControlFlowBuilder.cpp

```diff
@@ -287,6 +298,22 @@ bool ControlFlowBuilder::visit(FunctionCall const& _functionCall)
 				m_currentNode = nextNode;
 				return false;
 			}
+			case FunctionType::Kind::Internal:
+			{
+				m_currentNode->functionCalls.emplace_back(&_functionCall);
+				solAssert(m_revertNode, "");
+				visitNode(_functionCall);
+				_functionCall.expression().accept(*this);
+				ASTNode::listAccept(_functionCall.arguments(), *this);
+
+				connect(m_currentNode, m_revertNode);
```

Comment:
Why do you connect to the revert node?

Reply:
That is a very good question. I think I commented the code earlier and while cleaning up, this was left in. Luckily it had little effect on the behavior.